modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/SystemObjectRelatedObjectEntriesTest.java

```diff
@@ -273,6 +273,8 @@ public void testGetManyToOneSystemObjectRelatedObjectEntries()
 			"r_", objectRelationship.getName(), "_",
 			objectDefinitionNameReference);
 
+		// Default unrelated user
+
 		JSONObject jsonObject1 = HTTPTestUtil.invokeToJSONObject(
```

Comment:
As you can see, the pattern is always repeated:

```java
		JSONObject jsonObject1 = HTTPTestUtil.invokeToJSONObject(
			null,
			StringBundler.concat(
				_getLocation(), StringPool.SLASH, _user.getUserId()),
			Http.Method.GET);
		Assert.assertEquals(
			StringPool.BLANK,
			jsonObject1.get(relatedObjectEntryFieldBase + "ERC"));
		Assert.assertEquals(
			String.valueOf(0),
			jsonObject1.get(
				relatedObjectEntryFieldBase + "Id"
			).toString());
```

Maybe we can create a new method to extract and apply some part on the test:

```java

private void _testGetManyToOneSystemObjectRelatedObjectEntries(String expectedObjectEntryExternalReferenceCode, long expectedObjectEntryId,
		String relatedObjectEntryFieldBase, long userId) throws Exception {

		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
			null,
			StringBundler.concat(
				_getLocation(), StringPool.SLASH, userId),
			Http.Method.GET);

		Assert.assertEquals(
			expectedObjectEntryExternalReferenceCode,
			jsonObject.get(relatedObjectEntryFieldBase + "ERC"));
		Assert.assertEquals(expectedObjectEntryId,
			jsonObject.getLong(
				relatedObjectEntryFieldBase + "Id"
			));
	}
```